docs/guides/privileged_devices.md

```diff
@@ -0,0 +1,146 @@
+
+[Installation Guide](https://learn.microsoft.com/en-us/windows/wsl/install)
+
+Within the guide there is also a [recommended setup](https://docs.microsoft.com/en-us/windows/wsl/setup/environment#set-up-your-linux-user-info) of a development environment using WSL.
+
+This guide should also be followed as it includes setup of Docker Desktop, Git, Windows Terminal and VS Code.
+
+#### Windows permissions
+In order to allow WSL to change Windows file permissions, the drives need to be mounted with the metadata option.
+
+1. Create or edit WSL config file by entering the below terminal command.
+
+    ```
+    sudo nano /etc/wsl.conf
+    ```
+
+1. Add the following content to the file, then save and exit.
+
+    ```
+    [automount]
+    options = "metadata"
+    ```
+
+1. Restart WSL for changes to take effect.
+
+#### Proxy
+You will need to configure WSL to work around the proxy to avoid network conflicts.
+
+1. Create a `.wslconfig` file in your Windows profile directory (`C:\Users\<username>`) with the following content:
+
+    ```
+    [wsl2]
+    networkingMode=mirrored
+    autoProxy=false
+    ```
+
+1. Restart WSL for changes to take effect.
+
+#### Quick links
+- [Setup Docker Desktop](https://docs.microsoft.com/en-us/windows/wsl/tutorials/wsl-containers)
+- [Setup Git](https://docs.microsoft.com/en-us/windows/wsl/tutorials/wsl-git)
+- [Setup Visual Studio Code](https://docs.microsoft.com/en-us/windows/wsl/tutorials/wsl-vscode)
+- [Setup Windows Terminal](https://docs.microsoft.com/en-us/windows/terminal/get-started)
+
+### Docker Desktop
+You can install Docker Desktop from the Docker Web site:
+
+1. Download the [Docker Desktop installer](https://www.docker.com/products/docker-desktop) for your operating system.
+
+1. Run the installer and follow the prompts to install Docker Desktop.
+    - If you are using WSL ensure that `Use WSL 2 instead of Hyper-V (recommended)` is ticked during installation
+
+1. You will need to configure Docker Desktop to work around the proxy to avoid network conflicts. This can be done by right clicking the Docker Desktop icon in the system tray and selecting `Settings`. From there, select `Resources`, `Proxies` and ensure that `Manual proxy configuration` is checked.
+
+If you are running WSL:
+
+1. Check that Docker Desktop is set to use the WSL2 backend. This can be done by right-clicking the Docker Desktop icon in the system tray and selecting `Settings`. From there, select `General` and ensure `Use the WSL 2 based engine` is checked.
+
+1. Ensure that Docker Desktop is available to your WSL distro. This can be done by right-clicking the Docker Desktop icon in the system tray and selecting `Settings`. From there, select `Resources`, `WSL Integration` and ensure that the distro you are using is checked.
+
+Defra has a Docker organisation that you should join to ensure that you are licensed for your use of Docker Desktop.
```

Comment:
This point is relevant to other software we use too.  Should we add a section that covers development tool choice in general including a point to make sure that necessary licensing is understood before adoption?  Plus any data/monitoring etc.

Reply:
Added a section on licensing and another one specifically on security of downloaded software 👍